index.html

```diff
@@ -9,6 +9,7 @@
 <script src="parse-lexicon/greek-helpers.js"></script>
 <script defer src="parse-lexicon/AbbotSmithData.js"></script>
 <script src="parse-lexicon/AbbotSmithLexicon.js"></script>
+<script src="scripts/disambiguated_strongs.js"></script>

```

Comment:
For performance and clarity of intentions, we might want to eventually move these to `type="module"`

Per this:
https://www.w3schools.com/js/js_modules.asp

and this:

![image](https://github.com/user-attachments/assets/3be87c52-3425-4483-a879-17209c5d45c9)